explainaboard/loaders/file_loader.py

```diff
@@ -236,22 +236,26 @@ def add_sample():
                     field.src_name: [] for field in self._fields
                 }  # reset
 
+        max_field: int = max([narrow(x.src_name, int) for x in self._fields])
         for line in raw_data:
             # at sentence boundary
             if line.startswith("-DOCSTART-") or line == "" or line == "\n":
                 add_sample()
             else:
                 splits = line.split("\t")
-                if len(splits) < len(self._fields):  # not separated by tabs
+                if len(splits) <= max_field:  # not separated by tabs
                     splits = line.split(" ")
-                    if len(splits) < len(self._fields):  # not separated by spaces
-                        raise ValueError(
-                            f"not enough fields for {line} (sentence index: {guid})"
-                        )
+                if len(splits) <= max_field:  # not separated by tabs or spaces
+                    raise ValueError(
+                        f"not enough fields for {line} (sentence index: {guid})"
+                    )
 
                 for field in self._fields:
+                    fid = narrow(field.src_name, int)
+                    if fid >= len(splits):
```

Comment:
Thanks for the PR! It looks great! 

(Just one small thing) I'm trying to understand this check here. I think this may not be necessary since `len(splits) > max_field` is checked in the previous if/else block?

Reply:
Thanks, this was indeed not necessary given the checks above. It was something I had introduced in the process of debugging.